### ISSUE TITLE
Build fails on SDLmain due to undefined `WINBOOL` type

### ISSUE THREAD

**Issue by darkstar** [dosbox-staging/dosbox-staging#3359]
### Are you using the latest Dosbox-Staging Version?

- [X] I have checked releases and am using the latest release.

### Different version than latest?

git-d47e8c624d4303a98607746d86e51c5e5d52bf28

### What Operating System are you using?

Windows 10

### Have you checked that no other similar issue already exists?

- [X] I have searched and not found similar issues.

### A clear and concise description of what the bug is.

Compilation fails on MSVC 2022 Community (Version 17.8.5).

I get an error showing that somehow `WINBOOL` is not properly recognized as a type:
```
3>D:\Devel\dosbox-staging\src\gui\sdlmain.cpp(3169,23): warning C4244: 'argument': conversion from 'int' to 'const float', possible loss of data
3>D:\Devel\dosbox-staging\src\gui\sdlmain.cpp(3184,23): warning C4244: 'argument': conversion from 'int' to 'const float', possible loss of data
3>D:\Devel\dosbox-staging\src\gui\sdlmain.cpp(4568,16): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int
3>D:\Devel\dosbox-staging\src\gui\sdlmain.cpp(4568,16): error C2146: syntax error: missing ';' before identifier 'inherit_handles'
3>D:\Devel\dosbox-staging\src\gui\sdlmain.cpp(4568,16): error C2065: 'inherit_handles': undeclared identifier
3>D:\Devel\dosbox-staging\src\gui\sdlmain.cpp(4594,4): error C2065: 'inherit_handles': undeclared identifier
```

This is the line in question:
https://github.com/dosbox-staging/dosbox-staging/blob/main/src/gui/sdlmain.cpp#L4568
```
	const WINBOOL inherit_handles = FALSE;
```

After changing `WINBOOL` to just `BOOL`, dosbox-statging builds fine. According to the MSDN docs, the parameter should be of type BOOL anyway (there is no WINBOOL in MSDN)

But that commit apparently passed CI so it might be something on my end.

The relevant commit that introduced WINBOOL is 0d3c555cb16028face41c0546326503b0a409229

### Steps to reproduce the behaviour.

Explain how to reproduce
1. Install Visual Studio Version 17.8.5
2. Try to compile dosbox-staging
3. notice error

### Code of Conduct & Contributing Guidelines

- [X] Yes, I agree.

**Comment by weirddan455** [comment]
The fix is in #3358 which I’m going to merge shortly.  The MSVC build is semi unofficial right now.  It was removed from CI due to issues with the GitHub builders hanging up for hours.

**Comment by weirddan455** [comment]
Fix is merged.
